src/ui/public/timepicker/kbn_global_timepicker.html

```diff
@@ -21,6 +21,18 @@
     </span>
   </div>
 
+  <div class="localMenuItem" ng-click="zoomIn()">
+    <i class="fa fa-search-plus ng-scope" tooltip="Zoom in"></i>
```

Comment:
Instead of using the `<i>` element, can we use `<span>` everywhere here? It's more semantic markup.